### ISSUE TITLE
Social integration

### ISSUE THREAD

**Issue by asbjornenge** [meanwhile-news/meanwhile-app#6]
Først og fremst fjerne det "fake" greiene vi har nå, og så diskutere litt hvordan det bør være... Er det beste en facebook integrasjon? Blir vel en facebook app da som prater med et API hos oss...?

//cc @olekjormo @olavgg @megatrond

**Comment by megatrond** [comment]
Det vil kreve innloggingsfunksjonalitet også, siden brukeren må sende med et token av noe slag når de anbefaler ting.

Blir det node-proxyen som håndterer dette?

**Comment by asbjornenge** [comment]
Ja, men vi kunne kanskje kommet unna med en slik "Connect to facebook" greie... hvor vi mekker en konto og lagrer en cookie forever for dem... skulle den bli sletter så får de bare klikke "Connect to facebook" igjen...? Da har de jo egentlig en konto, bare de ikke vet om det :stuck_out_tongue:

Usikker om det hører hjemme i proxy egentlig... kunne kanskje vært en egen service?
